ooniprobe/View/RunTest/ProgressViewController.m

```diff
@@ -46,10 +46,23 @@ -(void)testStarted:(NSNotification *)notification{
 -(void)updateProgress:(NSNotification *)notification{
     NSDictionary *userInfo = notification.userInfo;
     NSNumber *prog = [userInfo objectForKey:@"prog"];
+    int totalRuntime = 0;
+    for (AbstractSuite *n in [RunningTest currentTest].iTestSuites) {
+        totalRuntime += n.getRuntime;
+    }
+
+    float previousProgress = 0;
+    for (AbstractSuite *n in [RunningTest currentTest].iTestSuites) {
+        if (![[RunningTest currentTest].testSuites containsObject:n]) {
+            previousProgress += n.getRuntime;
+        }
+    }
     float totalTests = [RunningTest currentTest].testSuite.totalTests;
     int index = [RunningTest currentTest].testSuite.measurementIdx;
     float prevProgress = index/totalTests;
+    float ratio = [RunningTest currentTest].testSuite.getRuntime/(float)totalRuntime;
```

Comment:
I would try to avoid division by zero by protecting against it with an `if` or a ternary operator